modules/imaging_browser/php/imagingbrowserrowprovisioner.class.inc

```diff
@@ -146,14 +149,14 @@ class ImagingBrowserRowProvisioner extends \LORIS\Data\Provisioners\DBRowProvisi
         foreach ($case_desc as $key => $value) {
             $left_joins .= "
             LEFT JOIN (
-              SELECT files.SessionID, 
-                MIN(files_qcstatus.QCStatus+0) as $qc[$key] 
-              FROM files 
-              JOIN files_qcstatus USING (FileID) 
-              WHERE files.AcquisitionProtocolID= $key 
-                AND files_qcstatus.QCStatus IN (1, 2) 
-              GROUP BY files.SessionID) $pass[$key] 
-                ON ($pass[$key].SessionID=f.SessionID
+              SELECT files.SessionID,
+                MIN(files_qcstatus.QCStatus+0) as $qc[$key]
+              FROM files
+              JOIN files_qcstatus USING (FileID)
+              WHERE files.AcquisitionProtocolID= $key
+                AND files_qcstatus.QCStatus IN (1, 2)
+              GROUP BY files.SessionID) `$pass[$key]`
+                ON (`$pass[$key]`.SessionID=f.SessionID
```

Comment:
Couldn't it be re-written like that instead of using backtic?

```suggestion
              GROUP BY files.SessionID) " . $DB->quote($pass[$key]) 
              . "ON (" . $DB->quote($pass[$key]) . ".SessionID=f.SessionID
```

Reply:
when I tried this, it returned the following error:

>[Fri Jun 05 09:51:14.495832 2020] [php7:warn] [pid 24948] [client ::1:56090] PHP Warning:  PDOStatement::execute(): SQLSTATE[42000]: Syntax error or access violation: 1064 You have an error in your SQL syntax; check the manual that corresponds to your MySQL server version for the right syntax to use near ''t1'ON ('t1'.SessionID=f.SessionID\n            ) \n            LEFT JOIN (\n      ' at line 73 in /var/www/loris/src/Data/Provisioners/DBRowProvisioner.php

I think it's because the single quotes without around a word without any back tics wrapped gets it to be interpreted as a string literal instead of column/table name. 
Also, in the latest commit, the single quotes are used on L115: https://github.com/aces/Loris/pull/6668/files#diff-246d74737579a11922903fcadb7015ffL112-R117 , so the back tics would be wrapped around the table name resulting in `` `'t1pass'` `` for example which would escape any special characters as well as for it to be interpreted as a table name.